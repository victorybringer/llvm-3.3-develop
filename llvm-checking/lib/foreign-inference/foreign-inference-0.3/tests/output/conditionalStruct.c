struct S {
  int f1;
  double f2;
};

extern struct S glob;

void f(struct S *out) {
  if(out) {
    *out = glob;
  }
}
