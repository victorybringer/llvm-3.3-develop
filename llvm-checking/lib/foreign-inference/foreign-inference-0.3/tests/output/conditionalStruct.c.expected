fromList [ ("@f", fromList [("%out", PAOut)]) ]
